commerce-test/src/testIntegration/java/com/liferay/commerce/internal/order/test/CommerceOrderHttpHelperImplTest.java

```diff
@@ -84,6 +87,20 @@
 
 	@Before
 	public void setUp() throws Exception {
+		List<CommerceInventoryBookedQuantity>
+			commerceInventoryBookedQuantities =
+				_commerceBookedQuantityLocalService.
+					getCommerceInventoryBookedQuantities(
+						QueryUtil.ALL_POS, QueryUtil.ALL_POS);
+
+		for (CommerceInventoryBookedQuantity commerceInventoryBookedQuantity :
+				commerceInventoryBookedQuantities) {
+
+			_commerceBookedQuantityLocalService.
+				deleteCommerceInventoryBookedQuantity(
+					commerceInventoryBookedQuantity);
+		}
```

Comment:
Oh dang!! This seems like.... kindof a nuclear option. Wouldn't the proper fix be to find wherever we're failing to clean up the `CommerceInventoryBookedQuantity` that a test created, and delete them in the cleanup of that test?

Reply:
Agree! @riccardo-alberti  please send a fix when you get time. Thanks